Sources/Kitura/TemplatingError.swift

```diff
@@ -17,6 +17,7 @@
 import Foundation
 
 enum TemplatingError: Swift.Error {
```

Comment:
I'm confused, isn't the default access level `internal`? How are we managing to throw this from a public method at the moment??